README.md

```diff
@@ -0,0 +1,44 @@
+# [Archivematica Storage Service](https://www.archivematica.org/)
+
+By [Artefactual](https://www.artefactual.com/)
+
+The Archivematica Storage Service is part of the Archivematica project.
+Archivematica is a web- and standards-based, open-source application which allows your institution to preserve long-term access to trustworthy, authentic and reliable digital content.
+Our target users are archivists, librarians, and anyone working to preserve digital objects.
+The Storage Service is responsible for moving files to and from Archivematica for processing and long-term storage.
+For more information, please see the [Archivematica documentation](https://www.archivematica.org/en/docs/).
+
+You are free to copy, modify, and distribute the Archivematica Storage Service with attribution under the terms of the AGPL license.
+See the [LICENSE](LICENSE) file for details.
+
+
+## Installation
+
+* [Production installation](https://www.archivematica.org/docs/storage-service/latest/)
```

Comment:
Our docs links are structured differently than AtoM's because we have multiple projects, this would work in the AtoM docs. @sevein do you have a suggestion?